src/Analyzers/CSharp/CodeFixes/ConvertToRecord/ConvertToRecordEngine.cs

```diff
@@ -775,8 +775,7 @@ private static SyntaxList<AttributeListSyntax> GetModifiedAttributeListsForPrope
                                             : token);
 
                                     if (index == 0 &&
-                                        tokens.Length >= 2 &&
-                                        tokens[0].IsKind(SyntaxKind.XmlTextLiteralNewLineToken))
+                                        tokens is [(kind: SyntaxKind.XmlTextLiteralNewLineToken), ..])
```

Comment:
This is not the same, its missing the `.Length >= 2` part of the check. I think you want:
```suggestion
                                        tokens is [(kind: SyntaxKind.XmlTextLiteralNewLineToken), _, ..])
```

Reply:
ooh, quite right.